state_of_html.yml

```
@@ -669,7 +669,7 @@ translations:
     t: What are your pain points around accessing and using native system and platform capabilities?
```

```suggestion
    t: What are your pain points around accessing and using native system capabilities?
```

(to avoid repetition)

Alternatively: 
```suggestion
    t: What are your pain points around accessing and using native OS or device capabilities?
```